### ISSUE TITLE
Any chance of continuation?

### ISSUE THREAD

**Issue by nikolal** [ramda/ramda-fantasy#161]
Is there chance that this project might continue updates in the future?

Maybe: sanctuary-js/sanctuary-maybe†
Either: sanctuary-js/sanctuary-either†
Future: fluture-js/Fluture
State: fantasyland/fantasy-states
Tuple: fantasyland/fantasy-tuples
Reader: fantasyland/fantasy-readers
IO: fantasyland/fantasy-io
Identity: fantasyland/fantasy-identities

As suggested in other thread, this is merged into sanctuary project, but their docs are awful, especially for someone new to functional programming.

PS

Thanks for making this in the first place.

**Comment by davidchambers** [comment]
This project has been retired, @nikolal. This [comparison][1] lists a number of features RF's Future type lacks. Other types are similarly lacking in functionality provided by other libraries. Additionally, RF still targets an old version of the Fantasy Land spec, so is incompatible with some newer libraries.

> [Sanctuary's] docs are awful, especially for someone new to functional programming.

I'd love suggestions as to how Sanctuary's documentation could be made more approachable. :)


[1]: https://github.com/fluture-js/Fluture/wiki/Comparison-of-Future-Implementations

**Comment by Avaq** [comment]
I think it's because RF's documentation (eg. [Either](https://github.com/ramda/ramda-fantasy/blob/master/docs/Either.md)) reads kind of like a tutorial. Maybe Sanctuary can include *working with the Either type* and *working with the Maybe type* pages somewhere (wiki, or `/docs`). Much of its content can be taken from here, if that's okay with @scott-christopher, of course. :)

**Comment by ferborva** [comment]
Indeed. A Wiki would be very nice! Also splitting the docs up would help.
My experience was also quite harsh. Both the github readme and the sanctuary org page are reaaaally long and I felt that reading top down killed the learning process. I'm sure that a better organization/splitting of the contents (types, utility functions, ...) would help navigating the docs. Put that together with a quick 'getting started' guide would bring more people into using it. Right now the docs are basically API definitions with a very very brief explanation for each item.

**Comment by davidchambers** [comment]
Thank you for the constructive feedback, @ferborva! sanctuary-js/sanctuary#210 has been at the back of my mind for quite some time, but I've been preoccupied with other Sanctuary improvements. I would love to find a way to sustain myself working on Sanctuary full-time, but it's currently a nights and weekends project.

**Comment by ferborva** [comment]
Completely understandable and a wonderful goal to point for ;)

**Comment by nikolal** [comment]
I agree with the posters above. I can give several suggestions. 
I will use Maybe type as an example:

1) Short textual explanation on Maybe, Just, Nothing etc.
2) Show core parts first. If someone new comes to docs and checks for Maybe he should first see the most important things, for example:
- S.fromMaybe
- S.toMaybe
- other important functions...
3)  Show simple example ( as current docs have ) with explanation
4) Show simple real life use case example 
5) At the end you can list the rest of the methods like the current docs show.

- Also it would be nice to have side menu instead of separate page, so users can navigate more easily.
- Styles update ( not really important, but it would help )

And also I agree with @ferborva suggestions about splitting docs and mini tutorial.
